org.leeroymanea.synology-note-station.json

```diff
@@ -0,0 +1,45 @@
+{
+  "app-id": "org.leeroymanea.synology-note-station",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+
+  "command": "/app/share/applications/org.leeroymanea.synology-note-station/launch.sh",
+  "finish-args": [
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--filesystem=host",
+    "--talk-name=org.freedesktop.Flatpak"
+  ],
+  "modules": [
+    {
+      "name": "synology-note-station",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "file",
+          "sha256":"a4c763bddda21bd211b0c8afdcff7e72a3edddc2392470fec8ce8123a0531751",
+          "url": "https://global.synologydownload.com/download/Utility/NoteStationClient/2.2.4-703/Ubuntu/x86_64/synology-note-station-client-2.2.4-703-linux-x64.deb"
```

Comment:
how is this gonna work on aarch64?

Reply:
also is this allowed to be redistributed since it doesn't look to be open source.